web/rest/client/config/routes.yaml

```diff
@@ -195,4 +195,14 @@ post_rating_profiles_simulate_call:
   defaults:
     _controller: Controller\Provider\SimulateCall\SimulateCallRatingProfileAction
     _api_item_operation_name: 'post_rating_profiles_simulate_call'
-    _api_receive: false
\ No newline at end of file
+    _api_receive: false
+
+post_holidays_dates_mass_import:
+  path: '/holiday_dates/mass_import'
+  methods: ['POST', 'GET']
```

Comment:
```suggestion
  methods: ['POST']
```

Reply:
If I remove this method, the api tests rises the next exception:
` No item route associated with the type \Model\\HolidaysMassImport\.`